src/app/containers/Include/index.test.jsx

```diff
@@ -157,4 +158,36 @@ describe('IncludeContainer', () => {
     expect(mockIdt2Amp).not.toHaveBeenCalled();
     expect(loggerMock.info).not.toHaveBeenCalled();
   });
+
+  it('should render a VJ include on an Amp page with toggles enabled', () => {
+    const vjProps = {
+      ampSrc: 'https://news.files.bbci.co.uk/vj.amp',
```

Comment:
This test data implies ampSrc ends `.amp` which I think is a bit misleading as it looks like this is page's path, not the include maybe:
```suggestion
      ampSrc: 'https://news.files.bbci.co.uk/include/something',
```
I tend to like to use real test data like here: https://github.com/bbc/simorgh/pull/7007/files#diff-754f58c9944064fefc2f77cd295825f9L444 , but that is a matter of preference.

I think if you want to use 'fake' test data it's better for it roughly represent the characteristics of the real data, e.g. not ending in `.amp`.